### PR TITLE
update code of conduct to v2.1

### PR DESCRIPTION
**Pull request by es183923**
more info: https://www.contributor-covenant.org/

### CODE REVIEW

**Comment by es183923** [comment]
idk i saw the other pr (but it was obv bot) so i thought maybe update to code of conduct's latest version?

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,74 +1,133 @@
...
+## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at support@brainbox.cc. All
-complaints will be reviewed and investigated and will result in a response that
-is deemed necessary and appropriate to the circumstances. The project team is
-obligated to maintain confidentiality with regard to the reporter of an incident.
-Further details of specific enforcement policies may be posted separately.
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (braindigitalis) [inline CODE_OF_CONDUCT.md:64]: insert the contact method 😂

Comment (es183923) [reply CODE_OF_CONDUCT.md:64]: what method do u want? discord?

Comment (braindigitalis) [reply CODE_OF_CONDUCT.md]: yeah discord is ok

**Pull request merged**